Added functional layouts for 'back-line' and 'spiral-line' pages (NEED images).

Sir-En00n3 opened the pull request:
'./src/app/practice/back-line/page.tsx'
and
'./src/app/practice/spiral-line/page.tsx'

 Both pages are placed as intended, pending required images."
Also removed misplaced and unnecessary file.
Added buttons to navigate from Home.

Sir-En00n3 committed:
Added functional layouts for pages:

'./src/app/practice/back-line/page.tsx'
and
'./src/app/practice/spiral-line/page.tsx'

 Both pages are placed as intended, pending required images."

shonhartman reviewed (commented):
This code looks good! You have a build error though. Looks like the types are not lining up for some reason. This will be a great exercise in debugging for you! Let me know if you need any help debugging it.

Sir-En00n3 committed:
Added functional layouts for pages:

'./src/app/practice/back-line/page.tsx'
and
'./src/app/practice/spiral-line/page.tsx'

 Both pages are placed as intended, pending required images."
